src/apps/SDIOutApp/SDIOutApp.js

```diff
@@ -46,6 +47,7 @@ class SDIOutApp extends Component {
                 })
                 .catch(err => {
                     console.error("[SDIOut] error fetching quad state", err);
+                    reportToSentry(`[SDIOut]: ${err}`, {err});
```

Comment:
Title is not informative, should be like console line in this case.
`"[SDIOut] error fetching quad state"`

extra data is ok.

Better if we have another method for reporting to sentry. Very similar to existing one but exceptions.
There is a difference between `captureException` and `captureMessage`